### ISSUE TITLE
The contents of http://packagist.org/p/providers-active.json do not match its signature

### ISSUE THREAD

**Issue by p** [composer/packagist#249]
We have at least two developers getting the following message:

"The contents of http://packagist.org/p/providers-active.json do not match its signature, this is most likely due to a temporary glitch but could indicate a man-in-the-middle attack. Try running composer again and please report it if it still persists."

Please investigate what is going on.

**Comment by bikashsapkota** [comment]
I got similar error and updated composer

the problem was solved

**Comment by RaazPuspa** [comment]
I got similar problem while doing `composer update`. Here's the error message I got:

`[Composer\Repository\RepositorySecurityException]
The contents of http://packagist.org/p/provider-2013%24282624b7b1179ec0d51dee126b20ab296da607a795c71f7b3a88aa60ec832857.json do not match its signature. This should indicate a man-in-the-middle attack. Try running composer again and report this if you think it is a mistake.`
